src/OpenTK/Graphics/GraphicsContext.cs

```diff
@@ -112,6 +129,18 @@ public GraphicsContext(GraphicsMode mode, IWindowInfo window, int major, int min
                 if (minor < 0)
                     minor = 0;
 
+                // Angle needs an embedded context
+                var use_angle_flag = GraphicsContextFlags.Angle
+                                | GraphicsContextFlags.AngleD3D9
+                                | GraphicsContextFlags.AngleD3D11
+                                | GraphicsContextFlags.AngleOpenGL;
+                var use_angle = false;
+                if ((flags & use_angle_flag) != 0)
```

Comment:
That's a totally reasonable point. The best explanation of the situation I can give is that OpenTK has been around for ages. Back when it first started, which is where a lot of this older code is from, it's very likely that the conventions for the library were different or just less rigorous.

I'm not sure at which point OpenTK adopted the guidelines, but it's pretty clearly stated in the [wiki's contributing page](https://github.com/opentk/opentk/wiki/Contributing).
Specifically, the current [Framework Design Guidelines](https://msdn.microsoft.com/en-us/library/ms229045(v=vs.110).aspx) state:
>X DO NOT use underscores, hyphens, or any other nonalphanumeric characters.

Thanks for your patience on this. I really do sympathise with the general annoyance of trying to meet an unclear convention in a PR.